src/types/my-cooperations/enums/myCooperations.enums.ts

```diff
@@ -0,0 +1,10 @@
+export enum ResourcesAvailability {
+  openAll = 'openAll',
+  openManually = 'openManually'
+}
+
+export enum ResourceAvailabilityStatus {
```

Comment:
Add Enum to the end of each enum: 
ResourcesAvailabilityEnum,
ResourceAvailabilityStatusEnum